src/android/res/drawable/ic_brand_logo.xml

```diff
@@ -0,0 +1,24 @@
.218,111.901C245.218,116.386 248.065,118.481 251.046,118.481C251.932,118.481 252.792,118.293 253.571,117.97C254.618,117.541 255.048,117.111 255.558,117.111C256.31,117.111 256.74,117.997 256.74,118.561C256.74,119.125 256.31,119.555 255.665,119.958C254.403,120.737 252.631,121.113 250.992,121.113C246.319,121.113 242.157,118.051 242.157,111.982C242.157,105.912 246.346,103.038 250.992,103.038C252.738,103.038 254.43,103.441 255.665,104.193C256.364,104.623 256.74,105.026 256.74,105.59C256.74,106.154 256.31,107.094 255.558,107.094C255.048,107.094 254.537,106.664 253.49,106.234C252.711,105.912 251.932,105.67 250.885,105.67C248.065,105.67 245.218,107.416 245.218,111.901ZM271.296,119.501L269.872,115.715L263.561,115.715L262.111,119.501C261.762,120.361 261.493,120.951 260.285,120.951C259.345,120.951 258.942,120.602 258.942,120.012C258.942,119.797 258.995,119.582 259.183,119.125L265.038,104.945C265.468,103.871 265.817,103.146 266.891,103.146C267.939,103.146 268.261,103.871 268.691,104.945L274.303,119.125C274.438,119.448 274.518,119.77 274.518,120.012C274.518,120.602 274.062,120.951 273.122,120.951C272.021,120.951 271.672,120.468 271.296,119.501ZM266.757,107.523L264.555,113.136L268.905,113.136L266.757,107.523ZM280.991,113.754L280.991,119.286C280.991,119.797 280.991,120.173 280.776,120.468C280.534,120.764 280.131,120.951 279.567,120.951C278.976,120.951 278.52,120.764 278.278,120.468C278.063,120.173 278.063,119.797 278.063,119.286L278.063,105.321C278.063,104.489 278.063,103.871 279.352,103.522C279.943,103.361 280.91,103.226 282.145,103.226C283.515,103.226 285.1,103.388 286.362,104.032C287.973,104.865 288.886,106.342 288.886,108.329C288.886,110.531 287.758,112.223 285.717,113.11L290.229,119.34C290.739,120.038 290.578,120.951 289.021,120.951C287.812,120.951 287.436,120.441 286.899,119.662L284.133,115.58C283.515,114.667 283.005,113.754 283.005,113.754C283.005,113.754 282.548,113.808 281.904,113.808C281.259,113.808 280.991,113.754 280.991,113.754ZM280.991,105.858L280.991,111.364C280.991,111.364 281.34,111.391 281.796,111.391C284.24,111.391 285.825,110.666 285.825,108.49C285.825,106.342 284.294,105.751 282.414,105.751C281.635,105.751 280.991,105.858 280.991,105.858ZM297.024,106.046L297.024,110.182L300.757,110.182C301.186,110.182 301.509,110.209 301.804,110.424C302.073,110.612 302.261,110.934 302.261,111.498C302.261,112.062 302.073,112.411 301.804,112.599C301.509,112.814 301.186,112.814 300.757,112.814L297.024,112.814L297.024,118.158L302.449,118.158C302.878,118.158 303.201,118.185 303.496,118.4C303.765,118.588 303.953,118.91 303.953,119.474C303.953,120.038 303.765,120.388 303.496,120.576C303.201,120.79 302.878,120.79 302.449,120.79L295.52,120.79C294.446,120.79 294.096,120.361 294.096,119.367L294.096,104.838C294.096,103.79 294.499,103.414 295.52,103.414L301.75,103.414C302.18,103.414 302.502,103.441 302.798,103.656C303.066,103.844 303.254,104.166 303.254,104.73C303.254,105.294 303.066,105.644 302.798,105.831C302.502,106.046 302.18,106.046 301.75,106.046L297.024,106.046Z"
```

Comment:
Can we move this to our cordova-build-override folder & use a generic image here?